modules/sdk-coin-apt/src/lib/transaction/transferTransaction.ts

```diff
@@ -50,4 +32,64 @@ export class TransferTransaction extends Transaction {
       feePayer: this.feePayerAddress,
     };
   }
+
+  fromRawTransaction(rawTransaction: string): void {
+    let signedTxn: SignedTransaction;
+    try {
+      signedTxn = utils.deserializeSignedTransaction(rawTransaction);
+    } catch (e) {
+      console.error('invalid raw transaction', e);
+      throw new Error('invalid raw transaction');
+    }
+    this.fromDeserializedSignedTransaction(signedTxn);
+  }
+
+  fromDeserializedSignedTransaction(signedTxn: SignedTransaction): void {
+    try {
+      const rawTxn = signedTxn.raw_txn;
+      this._sender = rawTxn.sender.toString();
+      this._recipient = utils.getRecipientFromTransactionPayload(rawTxn.payload);
```

Comment:
make `getRecipientFromTransactionPayload` an abstract method in the parent class.